language/move-prover/tests/sources/invariants.exp

```diff
@@ -23,43 +23,6 @@ error:  This assertion might not hold.
     =     at tests/sources/invariants.move:54:20: invalid_R_update
     =         r = <redacted>
 
-error:  This assertion might not hold.
```

Comment:
It's good that these errors disappear. However, I am not sure why these errors were there in the first place, and why they are disappearing in this PR.

Reply:
Why is it good that these errors are disappearing?

Reply:
Ah, just feeling good if there are less errors :) But, you're right. It's not necessarily a good thing. If the errors are valid, the errors messages should remain.

My PR removes the "// no-verify" lines, but added the "only-verify-spec" flag in testing. I am guessing that this have affected? (@wrwg)

Reply:
I did undo the flag change, and the errors came back. It turns out that these are important errors because the functions of the errors are incorrect and supposed to fail in verification.